consumer/nf_management_test.go

```diff
@@ -0,0 +1,53 @@
+package consumer_test
```

Comment:
License/Copyright header is missing
```suggestion
// SPDX-FileCopyrightText: 2024 Canonical ........
//
// SPDX-License-Identifier: Apache-2.0
//

package consumer_test
```